resources/js/components/shared/EllipsisMenu.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <b-dropdown
+    variant="ellipsis"
+    no-caret
+    no-flip
+    lazy
+    class="dropdown-right ellipsis-dropdown-main"
+  >
+    <template v-if="customButton" #button-content>
+      <i
+        class="pr-1 ellipsis-menu-icon"
+        :class="customButton.icon"
+      />
+      <span>{{ customButton.content }}</span>
+    </template>
+    <template v-else #button-content>
+      <i class="fas fa-ellipsis-h ellipsis-menu-icon" />
+    </template>
+    <div v-if="divider === true">
+      <b-dropdown-item
+        v-for="action in filterAboveDivider"
+        :key="action.value"
+        class="ellipsis-dropdown-item mx-auto"
+        @click="onClick(action, data)"
+      >
+        <div class="ellipsis-dropdown-content">
+          <i
+            class="pr-1"
+            :class="`${action.icon} fa-fw`"
```

Comment:
I would recommend adding `fa-fw` to the `class` line instead of the `:class` line so we don't have to do the interpolation on the `:class` line.

Reply:
Updated!